http/http-static/src/test/java/io/quarkus/ts/http/LargeStaticResourceIT.java

```diff
@@ -0,0 +1,41 @@
+package io.quarkus.ts.http;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.response.Response;
+
+@QuarkusTest
```

Comment:
This test is not using the test framework:
1.- Change it to `@QuarkusScenario`
2.- This annotation `@TestHTTPResource("big-file")` is not working in the test framework (you can file an issue to support it). Instead, use directly given().post("/big-file"),...